index.html

```diff
@@ -0,0 +1,89 @@
+      </div>
+      <img class="close" alt="close" src="./Images/close.png">
+      <img alt="logo" class="header-logo" src="./Images/logo.png">
+      <li class="links link1"><a class="menu-links" href="#home">Home</a></li>
+      <li class="links link2"><a class="menu-links" href="./about.html">About</a></li>
+      <li class="links link3"><a class="menu-links" href="#program">program</a></li>
+      <li class="links link3"><a class="menu-links" href="#program">Speakers</a></li>
+    </ul>
+  </section>
+  <section id="home" class="pg1-sec">
+    <h2 class="hello-world">"Hello, Sharing world"</h2>
+    <h1 class="event-name">The World Games <br>2022 Birmingham, USA</h1>
+    <p class="event-dis">The World Games is an 11-day international multi-sport event organized with the support of
+      the
+      International Olympic Committee. Held the year following the Summer Olympic Games, The World Games 2022 will
+      showcase a New Generation of Global Sports in Birmingham from July 7-17, 2022. An anticipated 3,600 elite
+      athletes
+      from over 100 countries will compete for gold in more than 30 of the fastest growing sports in the world.</p>
+    <p class="event-start">Birmingham, USA | July 7 - 17, 2022</p>
+  </section>
+  <section id="program" class="user-sec">
+    <div class="program">
+      <h1 class="program-h1">Program</h1>
+      <hr class="program-hr">
+    </div>
+    <div class="program-box">
+      <h4 class="date icon-hover"> <img alt="icon" class="icon" src="./Images/icons/icon-calendar.png">View Schedule
+        <p class="sub-text"> World Games is an 11-day international multi-sport event organized with the support of
+          the
+          International Olympic Committee</p>
+      </h4>
+      <h4 class="date icon-hover"> <img alt="icon" class="icon" src="./Images/icons/ticket.png">Buy Tickits Now
+        <p class="sub-text"> World Games is an 11-day international multi-sport event organized with the support of
+          the
+          International Olympic Committee</p>
+      </h4>>
+      <h4 class="date icon-hover"> <img alt="icon" class="icon" src="./Images/icons/icon-shoping.png">Buy TWG Gear <p
+          class="sub-text"> World Games is an 11-day international multi-sport event organized with the support of the
+          International Olympic Committee</p>
+      </h4>>
+      <h4 class="date icon-hover"> <img alt="icon" class="icon" src="./Images/icons/hearth.png">Day Pass <p
+          class="sub-text"> World Games is an 11-day international multi-sport event organized with the support of the
+          International Olympic Committee</p>
+      </h4>>
+    </div>
+  </section>
+  <section class="person1">
+    <h1 class="feacher-speakers">Featured Speakers</h1>
+    <hr class="speaker-hr1">
+    <div id="speaker" class= "persons"></div>
+  </section>
+  <button type="submit" class="more">MORE > </button>
+  <script src="./JS/index.js"></script>
+</body>
+
+</html>
```

Comment:
- [OPTIONAL] I have noticed that your Hope page misses the `Partner` section and `Footer` section 👇🏻 Kindly add those two missing parts 😜 

### These are only optional because due to oversight the previous reviewer probably missed them. However, they are extremely important. 💯 

- 
![25twomss](https://user-images.githubusercontent.com/79058364/175759229-850c857f-2978-4658-8d93-0b937867d1a2.png)